feat - made the home page

therajatg opened the pull request:
![16](https://user-images.githubusercontent.com/78013657/160147842-579ff4f6-649f-4ab1-8212-593832c09f20.gif)

therajatg committed:
feat - made the home page

rahulcs754 commented on my-app/src/frontend/pages/home/Home.js:
```diff
@@ -1,13 +1,19 @@
     <div>
-        <Navbar/>
-        <Products/>
+      <Navbar/>
```
you can use this navbar to app js.  
if you use navbar component in app js then you will not need to call every page

therajatg replied:
great suggestion man. The navabar and footer both can render only one time and not with every page. 

Made the suggested changes.

rahulcs754 reviewed (approved):
code looks good. 
Just Small suggestion
- remove extra space
- you can add nav component to app js . after that you will not need to import every component

Your design is looks good

therajatg committed:
refactor

added navbar and footer component in App.js so that they don't have to re-render with every page.

Mohitkumar33 commented on my-app/src/frontend/components/products/Products.js:
```diff
@@ -1,7 +1,44 @@
+    try {
+      const response = await axios.get("/api/products");
+      // const d = await response.json(); (If I do this then catch is shown in console)
+      console.log(response.data.products);
```
You can remove console.log

therajatg replied:
will do

Mohitkumar33 reviewed (approved):
code looks clean. You can use formatter (prettify) to beautify the code good. And you can change that in setting that auto format on save that will help you
